include/constants/event_object_var_values.h

```diff
@@ -0,0 +1,12 @@
+#ifndef GUARD_CONSTANTS_EVENT_OBJECT_VAR_VALUES_H
```

Comment:
I'd rename this to `event_var_values.h`.  And then add a comment above each set clearly stating which var the set is for.